eultheme/management/commands/end_downtime.py

```diff
@@ -0,0 +1,34 @@
+# Based on django-downtime downtime_end command
```

Comment:
Is it possible to keep the same command name, so that nothing needs to change when this gets fixed in django-downtime?  I don't know how django decides which one to use (order in installed apps?), but if it's something we can control that seems like something we could document.